setup.py

```diff
@@ -41,7 +41,7 @@
     author_email = 'pwang@enthought.com',
     maintainer = 'ETS Developers',
     maintainer_email = 'enthought-dev@enthought.com',
-    url = 'http://code.enthought.com/projects/chaco',
+    url = 'https://github.com/enthought/chaco'
```

Comment:
Apparently traitsui has had the url pointing to GitHub [since 2011](https://github.com/enthought/traitsui/commit/ab23cf1f4f739d76c3eff3f2ea3a074cdedeadbb).

We should at least be consistent and make a deliberate choice about it.